[2022-05-20 04:02:53,304] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: league_of_legends_dag.paths.path_maps.Create_StagingSnowflake_Tables_maps scheduled__2022-04-01T21:00:00+00:00 [queued]>
[2022-05-20 04:02:53,315] {taskinstance.py:1037} INFO - Dependencies all met for <TaskInstance: league_of_legends_dag.paths.path_maps.Create_StagingSnowflake_Tables_maps scheduled__2022-04-01T21:00:00+00:00 [queued]>
[2022-05-20 04:02:53,315] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-05-20 04:02:53,315] {taskinstance.py:1244} INFO - Starting attempt 1 of 2
[2022-05-20 04:02:53,315] {taskinstance.py:1245} INFO - 
--------------------------------------------------------------------------------
[2022-05-20 04:02:53,327] {taskinstance.py:1264} INFO - Executing <Task(SnowflakeOperator): paths.path_maps.Create_StagingSnowflake_Tables_maps> on 2022-04-01 21:00:00+00:00
[2022-05-20 04:02:53,333] {standard_task_runner.py:52} INFO - Started process 2928 to run task
[2022-05-20 04:02:53,336] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'league_of_legends_dag', 'paths.path_maps.Create_StagingSnowflake_Tables_maps', 'scheduled__2022-04-01T21:00:00+00:00', '--job-id', '575', '--raw', '--subdir', 'DAGS_FOLDER/dataDragon_dag.py', '--cfg-path', '/tmp/tmpeh14fxr4', '--error-file', '/tmp/tmpkkbaac7i']
[2022-05-20 04:02:53,337] {standard_task_runner.py:77} INFO - Job 575: Subtask paths.path_maps.Create_StagingSnowflake_Tables_maps
[2022-05-20 04:02:53,393] {logging_mixin.py:109} INFO - Running <TaskInstance: league_of_legends_dag.paths.path_maps.Create_StagingSnowflake_Tables_maps scheduled__2022-04-01T21:00:00+00:00 [running]> on host ca9a242fab95
[2022-05-20 04:02:53,458] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=league_of_legends_dag
AIRFLOW_CTX_TASK_ID=paths.path_maps.Create_StagingSnowflake_Tables_maps
AIRFLOW_CTX_EXECUTION_DATE=2022-04-01T21:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-01T21:00:00+00:00
[2022-05-20 04:02:53,459] {snowflake.py:118} INFO - Executing: CREATE OR REPLACE TABLE STAGING_DATADRAGON_MAPS (json_data variant);
[2022-05-20 04:02:53,472] {connection.py:257} INFO - Snowflake Connector for Python Version: 2.5.1, Python Version: 3.8.12, Platform: Linux-5.10.102.1-microsoft-standard-WSL2-x86_64-with-glibc2.2.5
[2022-05-20 04:02:53,473] {connection.py:868} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2022-05-20 04:02:53,474] {connection.py:886} INFO - Setting use_openssl_only mode to False
[2022-05-20 04:02:54,284] {cursor.py:637} INFO - query: [ALTER SESSION SET autocommit=True]
[2022-05-20 04:02:54,402] {cursor.py:661} INFO - query execution done
[2022-05-20 04:02:54,404] {snowflake.py:324} INFO - Running statement: CREATE OR REPLACE TABLE STAGING_DATADRAGON_MAPS (json_data variant);, parameters: None
[2022-05-20 04:02:54,404] {cursor.py:637} INFO - query: [CREATE OR REPLACE TABLE STAGING_DATADRAGON_MAPS (json_data variant);]
[2022-05-20 04:02:54,569] {cursor.py:661} INFO - query execution done
[2022-05-20 04:02:54,569] {snowflake.py:334} INFO - Statement execution info - {'status': 'Table STAGING_DATADRAGON_MAPS successfully created.'}
[2022-05-20 04:02:54,569] {json_result.py:127} INFO - fetching data done
[2022-05-20 04:02:54,569] {snowflake.py:338} INFO - Rows affected: 1
[2022-05-20 04:02:54,570] {snowflake.py:339} INFO - Snowflake query id: 01a46272-0000-f1b3-0000-d96900037522
[2022-05-20 04:02:54,570] {connection.py:499} INFO - closed
[2022-05-20 04:02:54,612] {connection.py:502} INFO - No async queries seem to be running, deleting session
[2022-05-20 04:02:54,717] {taskinstance.py:1272} INFO - Marking task as SUCCESS. dag_id=league_of_legends_dag, task_id=paths.path_maps.Create_StagingSnowflake_Tables_maps, execution_date=20220401T210000, start_date=20220520T040253, end_date=20220520T040254
[2022-05-20 04:02:54,754] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-05-20 04:02:54,810] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
